### PR TITLE
o [NXCM-4028] Add User Profile to Nexus

### PR DESCRIPTION
**Pull request by nabcos**
https://builds.sonatype.org/job/nexus-oss-its-feature/333/

### CODE REVIEW

**Comment by jdillon** [comment]
+1 lets get this merged to master

**Comment by adreghiciu** [comment]
+1 I just skimmed through but I can't see something to not go on. I guess you guys tested this by now for a while

File: nexus/nexus-webapp/src/main/webapp/js/Sonatype.view.js [hunk 1]
```diff
@@ -174,7 +174,73 @@ Sonatype.view = {
   // }
 };
 
+/**
+ * Very simple plugin for adding a close context menu to tabs.
+ *
```
Comment (peterlynch) [inline nexus/nexus-webapp/src/main/webapp/js/Sonatype.view.js:179]: This should have a jira to track this new feature. Please make one to get it in release notes.

**Comment by peterlynch** [comment]
+1

**Pull request merged**